test/molecule-role/molecule/secrets/Readme.md

```diff
@@ -0,0 +1,81 @@
+
+Secrets configuring
```

Comment:
please use proper markdown to make it a header

Reply:
reformatted